client/src/core/model-registry.js

```diff
@@ -274,6 +274,8 @@ class ModelRegistry {
       if (scale.factor) {
         obj3d.scale.copy(scale.factor);
         obj3d.getObjectByName(boundingBoxName).scale.copy(scale.factor);
+        obj3d.getObjectByName(boundingBoxName)
+            .material.color = new Color(0xff7F00);
```

Comment:
Interesting, but might as well get the bounding box only once, like `const boundingBox = obj3d.getObjectByName(boundingBoxName);` and then apply what needs to be applied there.